ArcadiaParties-SPA/src/app/user/user-identity/user-identity.component.scss

```diff
@@ -0,0 +1,14 @@
+.cursor {
+  cursor: pointer;
+  margin-left: 10px;
+}
+
+a {
```

Comment:
I see this class the second time. This is highly likely a reason to move it to common styles.

Reply:
Class moved into  common style.